bundle/src/main/java/com/adobe/acs/commons/httpcache/store/jcr/impl/writer/BucketNodeFactory.java

```diff
@@ -0,0 +1,65 @@
+package com.adobe.acs.commons.httpcache.store.jcr.impl.writer;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.jackrabbit.JcrConstants;
+import org.apache.jackrabbit.commons.JcrUtils;
+
+import com.adobe.acs.commons.httpcache.keys.CacheKey;
+import com.adobe.acs.commons.httpcache.store.jcr.impl.exceptions.BucketNodeFactoryException;
+
+/**
+ * Wrapper class for the bucket node
+ */
+public class BucketNodeFactory
```

Comment:
Could you add as a comment to this class an example cache key and how it is "serialized" to a node path?

And/or write up a test that shows this?

Reply:
I think you meant the EntryNodeWriter? Sure I can / should do that.